src/validate.js

```diff
@@ -0,0 +1,105 @@
+const axios = require('axios')
+const clone = require('lodash/clone')
+const pickBy = require('lodash/pickBy')
+const mapKeys = require('lodash/mapKeys')
+const isString = require('lodash/isString')
+const snakeCase = require('lodash/snakeCase')
+const config = require('./config')
+
+
+// Module API
+
+module.exports.validate = async (source, options) => {
```

Comment:
Adria it's the same as in `goodtables-py` and our API. Url or list of urls or list of datapackages (based on preset) etc. So it definitely requires perfect documentation :smiley: